### PR TITLE
new lab oop foundations

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,332 @@
...
+## Task #1 - Do not alter the `Athlete` class
+
+You will implement the following class hierarchy:
+
+- Every athlete has a jersey number.
+- Football players are athletes that make touchdowns.
+- Hockey players are athletes that score goals and get penalties.
+- An athlete's performance rating is calculated based on the type of athlete:
+    - The rating for a football player is calculated as (touchdowns * 5).
+    - The rating for a hockey player is calculated as (goals * 4 - penalties * 2).
```
Comment (kcleland0818) [inline README.md:24]: As a fan of sports, these ratings are rough! Lol! Football players also can get penalties too but I get it for the sake of the assignment haha.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,332 @@
...
+
+![task3 uml](https://curriculum-content.s3.amazonaws.com/6677/pillars/task3_uml.png)
+
+Create a new interface named `Flippable` with one abstract method named `flip()` that
+takes no parameters and has a `void` return type.
+
+Create a new class named `Coin` that implements `Flippable`.  The `flip()` method should
+print a random string of "Heads" or "Tails" each time it is called.
+
+Create a new class named `Mattrest` that implements `Flippable`. Add a boolean instance
```
Comment (kcleland0818) [inline README.md:292]: Typo: `Mattrest` should be `Mattress`

But I like how you tried to put the word "rest" in it haha

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md:24]: Made up some random calculations.  Let me know if there is something else that makes more sense.

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md:292]: :-)

**Comment by linda-seiter** [comment]
Breaking lab into 3 separate labs, 1 per task.

**Pull request closed without merging**